src/toxcore/dht/packet/dht_request.rs

```diff
@@ -368,6 +371,17 @@ impl ToBytes for DhtPkAnnouncePayload {
     }
 }
 
+impl DhtPkAnnouncePayload {
+    /// Create new `DhtPkAnnouncePayload` with `no_reply` set to current time.
+    pub fn new(dht_pk: PublicKey, nodes: Vec<TcpUdpPackedNode>) -> Self {
+        DhtPkAnnouncePayload {
```

Comment:
use std::time::SystemTime; here

Reply:
Then what about `use crate::toxcore::time::*;`?

Reply:
Right. That could be included too